hostinit/setup_test.go

```diff
@@ -50,8 +52,9 @@ func TestSetupReadyHosts(t *testing.T) {
 			h := hostsForTest[i]
 			So(h.Status, ShouldNotEqual, evergreen.HostRunning)
 		}
-		err := hostInit.startHosts(ctx)
-		So(err, ShouldBeNil)
+		// call it twice to get around rate-limiting
+		So(hostInit.startHosts(ctx), ShouldBeNil)
+		So(hostInit.startHosts(ctx), ShouldBeNil)
```

Comment:
Will one of these return an error though if we're rate limited?

Reply:
Ratelimiting just means that it returns before doing all the work. There's no error, but if there are ten mock hosts, and we only start 8, then we need to run this twice.